influxql/parser_test.go

```diff
@@ -2059,6 +2059,36 @@ func TestParser_ParseStatement(t *testing.T) {
 				ShardGroupDuration: 30 * time.Minute,
 			},
 		},
+		{
```

Comment:
Added these tests just for coverage.  This is just validating the parser allows for these values, and parses the durations correctly.  The integration tests then make sure the minimums are being enforced.